crates/ball_filter/src/hypothesis_merger.rs

```diff
@@ -0,0 +1,58 @@
+use crate::{hypothesis::BallHypothesis, BallFilter};
+
+pub struct RemovedHypotheses(pub(crate) Vec<BallHypothesis>);
+
+impl RemovedHypotheses {
+    pub fn inner(self) -> Vec<BallHypothesis> {
+        self.0
+    }
+}
+
+pub struct ValidHypotheses(Vec<BallHypothesis>);
```

Comment:
I think it simplifies the api since this type is returned from the update function. But of course I could make the removal a separated function instead

Reply:
not wrapped anymore